### PR TITLE
Add docker

### PR DESCRIPTION
**Pull request by AGuliaiev**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,6 +1,3 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/book_shop?createDatabaseIfNotExist=true
-spring.datasource.username=root
-spring.datasource.password=root
```
Comment (ahoienko) [inline src/main/resources/application.properties:3]: Please don`t push credentials to github

File: .env [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+MYSQL_PASSWORD=user
+MYSQL_ROOT_PASSWORD=superpassword
+MYSQLDB_USER=root
+MYSQLDB_PASSWORD=root
+MYSQL_LOCAL_PORT=3307
+MYSQL_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```
Comment (ahoienko) [inline .env]: Please don`t push credentials to github, you can left them empty or add placeholders

**Review by ahoienko — CHANGES_REQUESTED** [review]
Nice, left minor comments

File: src/main/resources/application.properties (hunk 1 above)
Comment (ahoienko) [inline src/main/resources/application.properties:3]: This properties still should be present, they just should be empty: spring.datasource.username= or replaced with placeholders or links to system variables spring.datasource.username={DB-USERNAME}

Comment (ahoienko) [reply src/main/resources/application.properties:3]: Same with env file

**Review by ahoienko — CHANGES_REQUESTED** [review]
Nice, but there are still some work to do

**Review by MentorMateAcademy — APPROVED** [review]

**Pull request merged**